crates/gitbutler-branch-actions/src/branch.rs

```diff
@@ -343,6 +349,17 @@ impl GroupBranch<'_> {
         }
         .map(BranchIdentity::from)
     }
+
+    /// Determines if the branch is a remote branch by ref name
+    fn is_remote_branch(&self, ref_name: &RemoteRefname) -> bool {
+        if let GroupBranch::Remote(branch) = self {
+            let remote_branch = ref_name.to_string();
```

Comment:
There should never be a need to convert `to_string()`. Maybe there is an `as_ref().as_bstr()` for that?

Reply:
Will update that 👍

Reply:
It still allocates, so I posted #5303 .

Reply:
Thanks!